modules/app_rentals.rb

```diff
@@ -0,0 +1,48 @@
+require './rental'
+
+module Rentals
+  def add_rental(books, people)
+    puts 'Select a book from the following list by number'
+    books.map.with_index { |book, index| puts "#{index}) Title: #{book.title}, Author: #{book.author}" }
+    book_index = gets.chomp.to_i
+    puts ''
+
+    puts 'Select a person from the following list by number'
+    people.map.with_index do |person, index|
+      puts "#{index}) [#{person.class}] Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+    person_index = gets.chomp.to_i
+    puts ''
+
+    print 'Date: '
+    date = gets.chomp
+
+    Rental.new(people[person_index], books[book_index], date)
+    puts 'Rental Created successfully'
```

Comment:
- [ ] in this method you are printing to the console, getting data from the user and saving the data as well, this is not in line with the single responsibility rules, please have another look at [this article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/lessons/solid.md#best-practices-and-solid) for more details.